docs/transpile/representing_quantum_computers.ipynb

```diff
@@ -5,9 +5,19 @@
    "id": "817782c5-8d62-492b-9633-6d57307e5d4b",
    "metadata": {},
    "source": [
-    "# Represent quantum computers\n",
+    "# Representing quantum computers for the transpiler\n",
```

Comment:
Nice change. We had a page in the old textbook called "Representing quantum computers" that was actually about the states of the computer rather than transpiler properties.

Reply:
Is it possible to still remove the -ing from "Representing"? Stylistically we're trying to keep consistency with directly active words (no -ing) but if there's a case to keep it, we of course can leave it as-is.

Reply:
The -ing makes sense to me mostly because using just "Represent" in the title reads as a verb and implies that we're going to do some action in order to represent quantum computers to the transpiler, rather than a passive description of how quantum computers are represented for the transpiler.  But I do understand the desire to keep everything stylistically consistent.